test/draw_test.py

```
@@ -1044,6 +1045,661 @@ class DrawEllipseTest(DrawEllipseMixin, DrawTestCase):
+    def test_aaellipse__args_missing(self):
+        """Ensures draw aaellipse detects any missing required args."""
+        surface = pygame.Surface((1, 1))
+
+        with self.assertRaises(TypeError):
+            bounds_rect = self.draw_aaellipse(surface, pygame.Color("red"))
+
+        with self.assertRaises(TypeError):
+            bounds_rect = self.draw_aaellipse(surface)
+
+        with self.assertRaises(TypeError):
+            bounds_rect = self.draw_aaellipse()
```

Flake8 (7.2.0)

[error] local variable 'bounds_rect' is assigned to but never used

(F841)